alecs-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,79 @@
+@import "partials/fonts";
+@import "partials/footer";
+@import "partials/header";
+@import "partials/layout";
+@import "partials/modules";
+@import "partials/user_icon";
+
+@import "partials/aspect_ratio";
+@import "partials/navbar_alec";
+
+@import "partials/portlet_asset_publisher";
+@import "partials/portlet_blogs";
+@import "partials/portlet_search";
+@import "portlet_decorator";
+
+body {
+	background-color: $main-bg-color;
+	position: relative;
+}
+
+textarea {
+	min-height: 50px;
+}
+
+#content {
+	background-color: #FCFCFC;
```

Comment:
How come some colors are listed in ```_aui_custom.scss``` and some are listed here?

Reply:
`_aui_custom.scss` wasn't created by me, probably from when I copied the theme. Dropping it and will look into other files that are unused.